pslab/external/hcsr04.py

```diff
@@ -0,0 +1,118 @@
+"""Ultrasonic distance sensors."""
```

Comment:
Should we have the file name in Uppercase?

Reply:
Python module names should be lowercase. https://peps.python.org/pep-0008/#package-and-module-names